docs/manual/src/bindings.md

```diff
@@ -13,6 +13,13 @@ Each of the bindings reads a file `uniffi.toml` in the root of a crate which sup
 various options which influence how the bindings are generated. Default options will be used
 if this file is missing.
 
+You can rename functions and structures via `[bindings.{language}.rename]` in the `uniffi.toml`. Options include: `enums`, `structs` and `functions`. For example:
+
+```toml
+[bindings.python.rename]
+enums.OtherError = "AnotherError"
+```
```

Comment:
I think I prefer the previous system where all the names were in one map, rather than splitting up by type.  There's no chance of name collision -- if there were, they're also be a collision in the Rust code.  Also, I think there could be some confusion since UniFFI has the concept of interfaces, which can be structs or enums.

Reply:
I agree on both sides... For now, it is too confusing because I think the generating of the `FFI` template uses structs and enums as type names. I liked the structured approach, it makes it very specific what will be changed. 

I think for now it is ok to remove it again. After we implmeneted your new issue with the more structured approach, we might want to revisit this imo.